external/netstandard20/netstandard20.depproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netstandard2.0;
```

Comment:
https://github.com/dotnet/corefx/blob/master/dir.props#L253 That is the place they get bin-placed to. I would keep this netstandard and if you like rename the project to match.